### DISCUSSION TITLE
Leveling Active resets after power off

### DISCUSSION THREAD

**Discussion by jxer** [Jyers/Marlin#725] [Q&A] [answered]
I am new to bed leveling. I am doing mesh till my bltouch comes in.

That said, every time i turn off the printer, it unchecks the leveling active option? is this expected? and what exactly does this do, if I have the m420 command in the slicer?

***actually I'm not sure if its a bug, If i start a print. the Bed leveling automatically checks after a print starts if i abort and go check it, even if i check it and uncheck it, then print, it rechecks itself?

what am I suppose to see here?

**Comment by Jyers** [comment, marked as answer]
That option is mostly for if you need to print something quick and don't want to add M420 to your start gcode. Otherwise you don't really need to worry about it.
